src/main/java/ru/practicum/shareit/item/model/CommentEntity.java

```diff
@@ -0,0 +1,44 @@
+package ru.practicum.shareit.item.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "comments", schema = "public")
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+public class CommentEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "comment_id", nullable = false)
+    private Long id;
+
+    @Column(name = "text", nullable = false)
+    private String text;
+
+    @Column(name = "item_id", nullable = false)
+    private Long itemId;
+
+    @Column(name = "author_id", nullable = false)
+    private Long authorId;
+
+    @Column(name = "created", nullable = false)
+    private LocalDateTime created;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof CommentEntity)) return false;
+        return id != null && id.equals(((CommentEntity) o).getId());
+    }
```

Comment:
методы equals и hashCode тоже можно было бы сделать через аннотции

Reply:
Я конечно сделал. Можно и через @Data @NoArgsConstructor. Но Code Analysis перед коммитом ругается: Warning:(12, 1) Using @EqualsAndHashCode for JPA entities is not recommended. It can cause severe performance and memory consumption issues. Впринципе я изначально потому и сделал через методы. Как тут правильно поступать?

Reply:
ну да, имеет место быть, тот варнинг который высвечивается в идее, 
https://www.jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/

так что, можешь вернуть обратно к виду с обычным переопределением, 
понятно, что сейчас в проекте ничего критичного не случится, но уже на реальной работе может быть все что угодно, лучше заранее иметь этот момент ввиду)